spraakbanken/flask-matomo2
# extend ci

- initial commit
- refactor: rename to check-check
- feat: add minimal and doctests jobs
- Bump actions/cache from 3 to 4
- initial commit
- fix: adjust names
- fix: make code more mergeable
- ci: fix so that nightly uses 3.13-dev
- ci: fix typo
- ci: add concurreny to test
- build: use pdm export
- ci(scheduled): install pdm separate
- ci: add check-for-updates
- ci(test): move codecov upload
- chore(test): add cov_report=xml
- ci(scheduled): bump update-deps-action
- test: rename lock file
- docs: add reason
- docs: add info about publish
- ci: remove check-for-updates
- ci: use right lockfile
- build: use pdm instead
- build: prepare import of common conf
- build: add project source
- ci: configure project
